cv32/sim/uvmt_cv32/Makefile

```diff
@@ -53,7 +53,7 @@ MKDIR_P = mkdir -p
 
 # Compile compile flags for all simulators (careful!)
 WAVES        ?= 0
-SV_CMP_FLAGS ?= "+define+CV32E40P_ASSERT_ON"
+#SV_CMP_FLAGS ?= "+define+CV32E40P_ASSERT_ON"
```

Comment:
Hmmm.  Why is this commented-out?

Reply:
Trying to run without a bunch of errors.

Reply:
I'll get rid of it in this PR.